### PR TITLE
feat(es/preset-env): Update preset-env data

### LINKED ISSUE #9544: Core-js forEach import polyfill not working, Babel with the same config works fine
### Describe the bug

SWC does not import forEach polyfill from core-js. Babel with the same config works fine (Import is present in the code: `require("core-js/modules/web.dom-collections.for-each.js");`).
Steps to reproduce:
1. Init npm and install dependencies `npm init && npm i -D @swc/cli @swc/core`
2. Create `index.js` entry file with following content
```JS
document.querySelectorAll('div').forEach(el => {
    console.log(el);
})
```
3. Create SWC config `.swcrc`:
```
{
    "$schema": "https://swc.rs/schema.json",
    "jsc": {
        "parser": {
            "syntax": "ecmascript",
        },
    },
    "env": {
        "targets": {
            "chrome": "40"
        },
        "mode": "usage",
        "coreJs": "3.22"
    }
}
```
4. Run: `npx swc index.js`
5. Check that forEach polyfill is not in imports

---

I prepared a comparison of SWC and BABEL playgrounds with the same config (env.mode = "usage"). You can see that BABEL includes a forEach polyfill for this code.

**SWC playground**: https://play.swc.rs/?version=1.7.24&code=H4sIAAAAAAAAA0vJTy7NTc0r0SssTS2qDE7NSU0uyS9yzMnRUE%2FJLFPX1EvLL3JNTM7QSCvNSy7JzM%2FTSM3RVKjmUgCC5Py84vycVL2c%2FHSQqDVXraY1AL32f8ZQAAAA&config=H4sIAAAAAAAAA1WOzQ6DIBCE7z4F2XPT9C899BGa9CEorhQjYlhoNMZ3ryBYvbHfDDMzFoxBTQIebJyf89FxS2jXeyY0tI73MwEUmpOwqnNwyGpNQap4QxjRtCjQGEOYlcS0alU1bLOF0Z1For0xWHkrG9wnFykdtCl9FNNmN3QY99Ed%2FqZctgaDolf%2B6axfGLbfTRC3El1YA%2BJjjUZ2O0Eeb8pY4olLzFAYi8%2Fovx4v50zfXlaqR0o9YVIx%2FQBKsvGpawEAAA%3D%3D

**Babel playground**: https://babeljs.io/repl#?browsers=chrome%2040&build=&builtIns=usage&corejs=3.21&spec=false&loose=false&code_lz=CYewxgrgtgpgdgFwHQEcIwE4E8DKMA2MYCIGAgvvgBQDkwAlgG40CUSAZqQKICGYAFlQIACALwA-YQG8AUMPnCwIOAGcQhJPhABzIfhYBuGQF8WQA&debug=false&forceAllTransforms=false&modules=false&shippedProposals=false&evaluate=false&fileSize=false&timeTravel=false&sourceType=script&lineWrap=true&presets=env&prettier=false&targets=&version=7.24.10&externalPlugins=&assumptions=%7B%7D

---

As you can see, babel adds the necessary import - `require("core-js/modules/web.dom-collections.for-each.js");`
When testing in chrome 40 (as in env.targets from .swcrc), the browser processes the babel code without errors, but the SWC code produces an error: `Uncaught TypeError: undefined is not a function`

### Input code

```typescript
document.querySelectorAll('div').forEach(el => {
    console.log(el);
})
```

### Config

```json
{
    "$schema": "https://swc.rs/schema.json",
    "jsc": {
        "parser": {
            "syntax": "ecmascript",
        },
    },
    "env": {
        "targets": {
            "chrome": "40"
        },
        "mode": "usage",
        "coreJs": "3.22"
    }
}
```

### Playground link (or link to the minimal reproduction)

https://play.swc.rs/?version=1.7.24&code=H4sIAAAAAAAAA0vJTy7NTc0r0SssTS2qDE7NSU0uyS9yzMnRUE%2FJLFPX1EvLL3JNTM7QSCvNSy7JzM%2FTSM3RVKjmUgCC5Py84vycVL2c%2FHSQqDVXraY1AL32f8ZQAAAA&config=H4sIAAAAAAAAA1WOzQ6DIBCE7z4F2XPT9C899BGa9CEorhQjYlhoNMZ3ryBYvbHfDDMzFoxBTQIebJyf89FxS2jXeyY0tI73MwEUmpOwqnNwyGpNQap4QxjRtCjQGEOYlcS0alU1bLOF0Z1For0xWHkrG9wnFykdtCl9FNNmN3QY99Ed%2FqZctgaDolf%2B6axfGLbfTRC3El1YA%2BJjjUZ2O0Eeb8pY4olLzFAYi8%2Fovx4v50zfXlaqR0o9YVIx%2FQBKsvGpawEAAA%3D%3D

### SWC Info output

Operating System:
        Platform: linux
        Arch: x64
        Machine Type: x86_64
        Version: #41-Ubuntu SMP PREEMPT_DYNAMIC Fri Aug  2 20:41:06 UTC 2024
        CPU: (16 cores)
            Models: Intel(R) Core(TM) i9-10980HK CPU @ 2.40GHz

Binaries:
    Node: 20.17.0
    npm: 
    Yarn: 0.32+git
    pnpm: N/A

Relevant Packages:
    @swc/core: 1.7.24
    @swc/helpers: N/A
    @swc/types: 0.1.12
        

### Expected behavior

The forEach polyfill import must be present in the code
`import "core-js/modules/web.dom-collections.for-each.js"`

### Actual behavior

Import forEach polyfill is currently not in the code

### Version

1.7.24

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by kdy1**
**Description:**

**Related issue:**

 - Closes https://github.com/swc-project/swc/issues/9544

### CODE REVIEW

File: crates/swc_ecma_preset_env/src/corejs3/builtin.rs [hunk 1]
```diff
@@ -173,506 +148,245 @@ static WEAK_SET_DEPENDENCIES: &[&str] = &concat2::<7>(
...
+    WeakMap: define("weak-map/index", WEAK_MAP_DEPENDENCIES),
+    WeakSet: define("weak-set/index", WEAK_SET_DEPENDENCIES),
+    clearImmediate: define("clear-immediate", ["web.immediate"]),
+    compositeKey: define("composite-key", ["esnext.composite-key"]),
+    compositeSymbol: define(
+        "composite-symbol",
+        ["esnext.composite-symbol", SYMBOL_DEPENDENCIES]
+    ),
+    fetch: define(null, PROMISE_DEPENDENCIES),
+    globalThis: define("global-this", ["es.global-this", "esnext.global-this"]),
```
Comment (kdy1) [inline crates/swc_ecma_preset_env/src/corejs3/builtin.rs:218]: The source of these new codes are https://github.com/babel/babel/blob/e498bee10f0123bb208baa228ce6417542a2c3c4/packages/babel-preset-env/src/polyfills/corejs3/built-in-definitions.js but I'm not sure if this is correct.

File: crates/swc/tests/fixture/issues-6xxx/6460/1/output/index.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 require("core-js/modules/es.array.at.js");
 require("core-js/modules/es.regexp.exec.js");
-require("core-js/modules/es.string.at-alternative.js");
+require("core-js/modules/es.string.at.js");
```
Comment (zloirock) [inline crates/swc/tests/fixture/issues-6xxx/6460/1/output/index.js:3]: `core-js` doesn't have a module with this name.

**Review by zloirock — CHANGES_REQUESTED** [review]

File: crates/swc_ecma_preset_env/src/corejs3/builtin.rs (hunk 1 above)
Comment (zloirock) [reply crates/swc_ecma_preset_env/src/corejs3/builtin.rs:218]: It's obsolete definitions. Actual definitions here https://github.com/babel/babel-polyfills/blob/main/packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts

File: crates/swc/tests/fixture/issues-9xxx/9544/output/1.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+require("core-js/modules/es.object.to-string.js");
+document.querySelectorAll('div').forEach(function(el) {
+    console.log(el);
+});
```
Comment (zloirock) [inline crates/swc/tests/fixture/issues-9xxx/9544/output/1.js:4]: @kdy1 to fix #9544, it should also load `web.dom-collections.for-each`, no?

Comment (kdy1) [reply crates/swc/tests/fixture/issues-9xxx/9544/output/1.js:4]: You are right. I'll reopen it.

**Pull request merged**